test/python/test_util.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2017, IBM.
```

Comment:
Nit: its 2018.

Reply:
Doh! Copyright came from old test_util.py I recreated from my branch after it was deleted from master. Looking through repo seems about 50/50 whether files have 2017 or 2018 in their copyright. Is that something worth changing?

Reply:
Yes, it is. If the file is new, we must provide the year of creation. Sorry for blocking on this but it's necessary.

Reply:
Oh, I'd already made the changes for `test_util.py`. I meant more in general. You answered my question either way!